server/admin_management/api/app/services/access_control_privilege_service.py

```diff
@@ -234,6 +232,7 @@ def send_email_notification(
                 return
```

Comment:
do we want to return the status not required in this case?

Reply:
That should be good, the default is not_required in the variable.